portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase

```diff
@@ -1583,7 +1579,7 @@ definition {
 
 			ACDXPSettings.goToInstanceSettingsAC();
 
-			ACDXPSettings.connectAnalyticsCloud();
```

Comment:
Hey @joshitagaki, `Site Name` is not created in this test, so we should use `ACDXPSettings.connectAnalyticsCloud();`